test/integration/binaries/test_binaries.py

```diff
@@ -59,7 +59,7 @@ def test_binaries_no_directory(tt_cmd, tmpdir):
 
 def test_binaries_empty_directory(tt_cmd, tmpdir):
     configPath = os.path.join(tmpdir, config_name)
-    os.mkdir(tmpdir+"/bin")
+    os.mkdir(tmpdir + "/bin")
```

Comment:
that change doesn't belong to the commit :)

Reply:
Fixed.